docker-compose.yaml

```
@@ -10,16 +10,13 @@ services:
       NODE_ENV: production
       HOST: '${BE_HOST}'
       PORT: '${BE_PORT}'
-      MONGO_URL: 'mongodb://db:27017'
-      MONGO_DATABASE: '${MONGO_DATABASE-tm-ks}'
+      MONGO_HOST: 'mongodb'
     volumes:
       - ./be:/home/node/app
     ports:
       - '8080:8080'
     command: 'npm run dev:start'
-  mongo:
+  mongodb:
```

Please keep the naming
```suggestion
  mongo:
```